tests/reports/data/tests-endpoint-registry.php

```diff
@@ -0,0 +1,433 @@
+<?php
+namespace EDD\Reports\Data;
+
+require_once EDD_PLUGIN_DIR . 'includes/admin/reporting/reports.php';
+
+if ( ! class_exists( '\EDD\Reports\Init' ) ) {
+	require_once( EDD_PLUGIN_DIR . 'includes/reports/class-init.php' );
+}
+
+/**
+ * Tests for the Endpoint registry API.
+ *
+ * @group edd_registry
+ * @group edd_reports
+ * @group edd_reports_endpoints
+ */
+class Endpoint_Registry_Tests extends \EDD_UnitTestCase {
+
+	/**
+	 * Reports fixture.
+	 *
+	 * @var \EDD\Reports\Init
+	 * @static
+	 */
+	protected static $reports;
+
+	/**
+	 * Endpoint registry fixture.
+	 *
+	 * @access protected
+	 * @var    \EDD\Reports\Data\Endpoint_Registry
+	 */
+	protected $registry;
+
+	/**
+	 * Set up fixtures once.
+	 */
+	public static function wpSetUpBeforeClass() {
+		self::$reports = new \EDD\Reports\Init();
+	}
+
+	/**
+	 * Set up fixtures once.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->registry = new \EDD\Reports\Data\Endpoint_Registry();
+	}
+
+	/**
+	 * Runs after each test to reset the items array.
+	 *
+	 * @access public
+	 */
+	public function tearDown() {
+		$this->registry->exchangeArray( array() );
+
+		parent::tearDown();
+	}
+
+	/**
+	 * @covers \EDD\Reports\Data\Endpoint_Registry::$item_error_label
+	 */
+	public function test_item_error_label_should_be_reports_endpoint() {
+		$this->assertSame( 'reports endpoint', $this->registry::$item_error_label );
```

Comment:
`$this->registry::$item_error_label` breaks the unit tests, as it's only available on PHP 7+.

Reply:
Out of curiosity, what's the error message? I don't see anything about this that would be PHP 7 specific.

Reply:
```
Parse error: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in /home/travis/build/easydigitaldownloads/easy-digital-downloads/tests/reports/data/tests-endpoint-registry.php on line 66
```